makefiles/pidof.mk

```diff
@@ -0,0 +1,46 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+STRAPPROJECTS += pidof
```

Comment:
This is a `SUBPROJECT` not a `STRAPPROJECT`

Reply:
All right